docs/wiki/Home.md

```diff
@@ -1,6 +1,9 @@
 # Common Azure Resource Modules Library
 
-The objective of this repository is to provide a template library that can accelerate deployments in Infrastructure as Code scenarios, such as Azure landing zones, landing zone accelerators and aspiring landing zone accelerators (workloads/applications/multi-module solutions) or individual service deployments.
+CARML resource library provides [bicep](https://github.com/Azure/bicep) modules which can be re-used in your infrastructure as code project to accelerate deployments. You can re-use code library as whole or in part. The primary aim is to provide you with re-usable infrastructure pattern so that you can focus what matter the most.
```

Comment:
```suggestion
The CARML resource library provides [bicep](https://github.com/Azure/bicep) modules which can be re-used in your infrastructure as code project to accelerate deployments. You can re-use the code library as whole or in part. The primary aim is to provide you with re-usable infrastructure pattern so that you can focus what matter the most.
```